### PR TITLE
Sign in form 

### PR DESCRIPTION
**Pull request by AhmedMughessib**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,13 +1,17 @@
-import { Box } from '@mui/material';
+// import { Box } from '@mui/material';
```
Comment (AdhamHaisami) [inline src/App.jsx:1]: delete comments

File: src/App.jsx [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+/* eslint-disable linebreak-style */
 import './App.css';
+import SignInComp from './components/singinform';
+
 import THEME from './utilize/Theme';
-import { SideBar } from './components';
+// import { SideBar, MenuComp } from './components';
```
Comment (AdhamHaisami) [inline src/App.jsx:7]: from here also

File: src/App.jsx [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
 
 const App = () => (
   <THEME>
-    <Box>
+    {/* <Box>
       <SideBar />
```
Comment (AdhamHaisami) [inline src/App.jsx:12]: delete comment and imported component from app

File: src/components/inputfield.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { TextField, InputAdornment } from '@mui/material';
+
+const InputComp = (props) => {
+  const { name, icon, type } = props;
+
+  let height;
+  let width;
+  if (type === 'signin') {
```
Comment (AdhamHaisami) [inline src/components/inputfield.jsx:9]: no need for an if statement, pass it as a class

File: src/components/inputfield.jsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    height = '80.69px';
+    width = '671.76px';
```
Comment (AdhamHaisami) [inline src/components/inputfield.jsx:11]: dimensions must be in rem not px

**Review by AdhamHaisami — APPROVED** [review]

**Review by HamdyPs — APPROVED** [review]

**Review by AdhamHaisami — APPROVED** [review]

**Review by heleena-fadi — APPROVED** [review]

**Review by AdhamHaisami — APPROVED** [review]

**Pull request merged**